infra/src/dsl/bff.ts

```diff
@@ -29,17 +44,24 @@ export const bffConfig = ({
   clientName,
   clientId,
   globalPrefix,
-  allowedRedirectUris,
 }: BffInfo) => {
   const sanitizeGlobalPrefix = sanitizePath(globalPrefix)
 
-  const getBaseUrl = (ctx: Context) =>
-    ctx.featureDeploymentName
+  const getBaseUrl = (ctx: Context) => {
+    const domain = ctx.featureDeploymentName
       ? `${ctx.featureDeploymentName}-beta.${ctx.env.domain}`
       : ctx.env.type === 'prod'
       ? ctx.env.domain
       : `beta.${ctx.env.domain}`
 
+    return `https://${domain}`
+  }
+
+  const getRedirectUris = (baseUrl: string, key: PortalKeys) => [
+    baseUrl,
+    ...(key === MINAR_SIDUR ? [`${baseUrl}/umsoknir`] : []),
+  ]
```

Comment:
This still returns `baseUrl` as an allowed redirect uri. Should only allow baseUrl/minarsidur + baseUrl/umsoknir or baseUrl/stjornbord.